docs/samples/explanation/aix/Fetch_20newsgroups/rfserver/setup.py

```diff
@@ -0,0 +1,40 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import setup, find_packages
+
+tests_require = [
+    'pytest',
+    'pytest-tornasync',
+    'mypy'
+]
+setup(
```

Comment:
Do we still want this setup file as the rfserver is no longer needed ?

Reply:
Let's delete this file

Reply:
@yuzisun I've deleted the file, thanks